nextcord/client.py

```diff
@@ -2869,7 +3001,38 @@ class be provided, it should be a subclass of :py:class:`Interaction`.
         .. note::
 
             This is synchronous due to how slash commands are implemented.
-        """
+
+        Warns
+        --------
+        ~FutureWarning
+            Warns when the cls parameter is redefined to a custom class. This serves as a warning
+            in the event that defining this parameter was a mistake, and other parameters were to be defined instead.
+        """
+        # check if they are creating custom interaction subclasses
+        if cls != Interaction:
```

Comment:
```suggestion
        if cls is not Interaction:
```
`!=` is usually used with literals.